app/models/lesson.rb

```diff
@@ -31,6 +31,21 @@ class Lesson < ActiveRecord::Base
     default_url: ""
   validates_attachment_content_type :overview, content_type: ["application/pdf"]
 
+  has_attached_file :first_photocopy,
+    styles: {thumbnail: "60x60#"},
+    default_url: ""
+  validates_attachment_content_type :first_photocopy, content_type: ["application/pdf"]
+
+  has_attached_file :second_photocopy,
+    styles: {thumbnail: "60x60#"},
+    default_url: ""
+  validates_attachment_content_type :second_photocopy, content_type: ["application/pdf"]
+
+  has_attached_file :third_photocopy,
+    styles: {thumbnail: "60x60#"},
+    default_url: ""
+  validates_attachment_content_type :third_photocopy, content_type: ["application/pdf"]
+
```

Comment:
@jasonoverby This def works for now, but we should talk about getting a more dynamic system for attaching assets sooner than later.